src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+  background-position: center;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
remove `margin-right: 4px;` from `.stars__star` 
and  use `:not(:last-child)`
Example:
```
.stars__star:not(:last-child) {
  margin-right: 4px;
}
```